dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
   -ms-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+/* container css for most high level containers  */
+.container-parent {
+  height: 100%;
+  position: relative;
+}
+
+.container-header {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+}
+
+.container-navSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-bottom: 4%;
+}
+
+.container-nav {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 5%;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.container-moto {
```

Comment:
added letter spacing of 4px. Adjusted the text to be centered when I addressed one of the first points above about using a `<p>` tag instead of a `<span>` or `<div>`. Text is centered but the letter spacing may be more than you're looking for here.